docs/vyos.vyos.vyos_lldp_interfaces_module.rst

```diff
@@ -397,18 +399,19 @@ Examples
                 longitude: 222.267255W
                 latitude: 33.524449N
         state: merged
-
     #
     #
     # -------------------------
     # Module Execution Result
     # -------------------------
     #
-    # before": []
+    # before": {}
     #
     #    "commands": [
     #        "set service lldp interface eth1 location civic-based country-code 'US'",
     #        "set service lldp interface eth1 location civic-based ca-type 0 ca-value 'ENGLISH'",
+    #        "set service lldp interface eth1 location civic-based country-code 'US'",
```

Comment:
This syntax does not exist `location civic-based`
```
vyos@client-11# set service lldp interface eth1 location 
Possible completions:
 > coordinate-based
                Coordinate based location
   elin         ECS ELIN (Emergency location identifier number)

      
[edit]
vyos@client-11# run show ver

Version:          VyOS 1.3.8
Release train:    equuleus

```

Reply:
`LLDP` module hasn't been updated for 1.3.8, as those changes happen, we'll be updating these items as part of the testing review. This module is consistent with 1.2.